docs/basics/installation.rst

```diff
@@ -46,41 +91,205 @@ You can find detailed installation instructions on how to do this
+      - Check for Python in your start menu. If no search results
+        show up, go to https://www.python.org/downloads.
+        The most recent version of Python is on the "Download"
+        button near the top of the page. Click on ``download``,
+        then ``save``.
 
-   $ brew install git-annex
+      - Run the Python installer by double-clicking the downloaded
+        ``python-<version>.exe`` file in your Downloads or clicking
+        on the Python 3.x result in your start menu.
+
+      - **Check the box "Add Python <version> to PATH** at the bottom
+        of the window, and select "Customize installation".
+
+      - Stay with the default options to install all optional features,
+        and additionally tick
+        "Add Python to environment variables" on the second page.
+        Optionally, tick the box
+        "Install for all users" to ensure that other users on the computer
+        are able to use Python.
+
+      - Finish the installation. After successful installation, instruct the installer
+        to bypass the 260 character file path limit (option available at the bottom
+        of the window).
+
+      - Check the installation by opening CMD (type ``cmd`` into the Windows
+        search bar and press ``Enter``) and typing python + ``Enter``. You should
+        see Python start up in the terminal. This means that Python is working
+        and the ``PATH`` is set correctly. Yay!
 
-Once Git-annex is available, DataLad can be installed via ``pip`` as described above.
-``pip`` comes with Python distributions, such as `anaconda <https://www.continuum.io/downloads>`_.
+   - **Step 2**: Install Git
 
+      - Go to https://git-scm.com/downloads, select Windows, and
+        **download the 32-bit Git for Windows Setup** (not the 64-bit version!)
+
+   - **Step 3**: Install Git-annex
+
+      - Obtain the current Git-annex versions installer
+        `from here <https://downloads.kitenet.net/git-annex/windows/current/>`_.
+        Save the file, and double click the downloaded
+        ``git-annex-installer.exe`` in your Downloads.
+
+      - During installation, you will be prompted to "Choose Install Location".
+        **Specify the directory in which Git is installed**.
+
+   - **Step 4**: Install DataLad via pip
+
+      - ``pip`` should be installed together with recent Python versions on
+        Windows. Open ``cmd`` and type ``pip install --user datalad``.
+
+
+
+.. container:: toggle
+
+   .. container:: header
+
+      **2) Install within WSL [recommended]**
+
+   The Windows Subsystem for Linux (WSL) allows Windows users to have full access
+   to a Linux distribution within Windows.
+   The improves the DataLad experience on Windows *greatly*.
+
+   If you have always used Windows be prepared for some user experience changes when
+   using Linux compared to Windows. For one, there will be no graphical user interface
+   (GUI). Instead, you will work inside of the standard Linux terminal. This however
```

Comment:
This sounds scary, but it only refers to the terminal window with datalad inside, no?